packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -1502,6 +1502,12 @@ spec:
       - op: add
         path: /spec/template/spec/joinConfiguration/nodeRegistration/kubeletExtraArgs/register-with-taints
         value: os=windows:NoSchedule
+      - op: add
+        path: /spec/template/spec/files/-
+        value:
+          path: 'C:\k\prevent_windows_updates.ps1'
+          content: |
+            Set-Service -Name "wuauserv" -StartupType Disabled -Status Stopped
```

Comment:
should this also be in `providers/infrastructure-vsphere/v1.4.1/cconly/base.yml`